compute_tools/src/installed_extensions.rs

```diff
@@ -42,75 +41,51 @@ fn list_dbs(client: &mut Client) -> Result<Vec<String>> {
 ///
 /// Same extension can be installed in multiple databases with different versions,
 /// we only keep the highest and lowest version across all databases.
-pub async fn get_installed_extensions(connstr: Url) -> Result<InstalledExtensions> {
```

Comment:
This is just incidental refactoring, right? Maybe split off to a separate PR

Reply:
Yeah, but it was so hard to read, so I decided to just do it as well. It's a bit hard to separate now, so let's keep it altogether